content/guides/getting-started.md

```diff
@@ -109,7 +109,7 @@ __src/index.js__
 +
   function component() {
     var element = document.createElement('div');
-  
+
 -   // Lodash, currently included via a script, is required for this line to work
```

Comment:
@TheDutchCoder just double-checked this. It's a bit confusing because your viewing a code block of the syntax highlighting language `diff` within a GitHub change diff. It tripped me up too when I first saw your comment. The diff from source tree is easier to understand:

![image](https://user-images.githubusercontent.com/8988697/27562694-b844270c-5a9b-11e7-8fc0-d95d3ac2b20f.png)

It's only the blank line above that comment that changed. Just a minor editorconfig autofix 👍 . Your comment should remain untouched.